libs/api/src/services/admin/types.ts

```diff
@@ -485,25 +484,13 @@ export const WriteInSummaryEntryNonPendingSchema: z.ZodSchema<WriteInSummaryEntr
 /**
  * Write-in image information.
  */
-export interface WriteInImageEntry {
+export interface WriteInImageView {
```

Comment:
I like `View` more than `Entry`! Maybe worth updating the comment for this interface to make it even clearer what it represents?